wire-ios/Wire-iOS/Sources/UserInterface/Settings/MFMailComposeViewController+Logs.swift

```diff
@@ -23,6 +23,45 @@ import WireCommonComponents
 
 extension MFMailComposeViewController {
 
+    func prefilledBody(withMessage message: String = "") -> String {
+        let date = Date()
+        let device = UIDevice.current.zm_model()
+
+        var body = """
+        --DO NOT EDIT--
+        App Version: \(Bundle.main.appInfo.fullVersion)
+        Bundle id: \(Bundle.main.bundleIdentifier ?? "-")
+        Device: \(device)
+        iOS version: \(UIDevice.current.systemVersion)
+        Date: \(date.transportString())
+        """
+
+#if DATADOG_IMPORT
+        // datadogId has always a value. NONE by default
+        // display only when enabled
+        if let datadogId = DatadogWrapper.shared?.datadogUserId {
+            body.append("\nDatadog ID: \(datadogId)")
+        }
+#endif
+        body.append("\n---------------\n")
+        typealias l10n = L10n.Localizable.Self.Settings.TechnicalReport.MailBody
+        let details = """
+        \(l10n.firstline)
+
+        - \(l10n.section1)
+
+
+        - \(l10n.section2)
+        \(message)
+
+        - \(l10n.section3)
+
+
+        """
+        body.append("\n\(details)\n")
```

Comment:
Now I've seen this in a test mail recently:
<img width="380" alt="Screenshot 2024-02-20 at 09 44 52" src="https://github.com/wireapp/wire-ios/assets/7196958/335f96fe-eedb-4589-bcbe-d65f439b9f8b">

For me as a user the big DO NOT EDIT is first and the eye read from top to bottom. Can we change the order in this way to have it more user friendly?
Maybe we can even remove the explicit mention "DO NOT EDIT"?

```
Hello, please fill in the following:

- Date and Time of the issue occured:


- What Happened:


- Steps to reproduce (if relevant):


---------------

--DO NOT EDIT--
App Version: 3.111.8 (999999)
Bundle id: com.wearezeta.zclient.alpha
Device: iPhone 14 Pro
iOS version: 17.2.1
Date: 2024-02-19T14:59:04.346Z
```

Reply:
I'd rather keep the same for now, this is copy from Android. But good input, can you create a ticket for it?

Reply:
see https://wearezeta.atlassian.net/browse/WPB-6768

Reply:
Ups I did this now too https://wearezeta.atlassian.net/browse/WPB-6769, let's close one